test/integration/consul-container/libs/assert/envoy.go

```diff
@@ -39,14 +41,11 @@ func GetEnvoyListenerTCPFilters(t *testing.T, adminPort int) {
 	filter := `.configs[2].dynamic_listeners[].active_state.listener | "\(.name) \( .filter_chains[0].filters | map(.name) | join(","))"`
 	results, err := utils.JQFilter(dump, filter)
 	require.NoError(t, err, "could not parse envoy configuration")
-
-	if len(results) != 2 {
-		require.Error(t, fmt.Errorf("s1 proxy should have been configured with one rbac listener filter. Got %d listener(s)", len(results)))
-	}
+	require.Len(t, results, 2, "static-server proxy should have been configured with two listener filters")
 
 	var filteredResult []string
 	for _, result := range results {
-		santizedResult := sanitizeResult(result)
+		santizedResult := libservice.SanitizeResult(result)
```

Comment:
Shall we keep `sanitizeResult` in this package since it's not being used elsewhere.